client/src/redux/ga-saga.js

```diff
@@ -31,7 +40,9 @@ function* callGaType({ payload: { type, data } }) {
         // Test_Type
         dimension4: aBTestConfig.type,
         // Test_Variation
-        dimension5: emailToABVariant(email).isAVariant ? 'A' : 'B'
+        dimension5: emailToABVariant(email).isAVariant ? 'A' : 'B',
+        // View_Type
+        dimension6: viewType
```

Comment:
Does it matter if you send a `null` value for this?

Reply:
not really. we could replace values for our analytics preview and at the moment only modal has different modal variations.